### ISSUE TITLE
Detect face id sensor working or not without enrolling

### ISSUE THREAD

**Issue by Johnny314** [gilshahar7/PearlRetry#2]
is it possible to detect face id sensors is working or not with Pearl Retry?

**Comment by gilshahar7** [comment]
Umm I’m not sure I understand your question, can you rephrase?

**Comment by Johnny314** [comment]
i want enroll face id with in my app and for that purpose i am loading BiometricUIKit framework(i know that private framework app will be rejected but i am creating this app for in house distribution)
In BiometricUIKit framework i am loading BKUIPearlEnrollController.h but it throw exception when i click get started button 
![face1](https://user-images.githubusercontent.com/42666898/62028933-edbc6d00-b1fa-11e9-8cd1-8d54428dd512.jpeg)
![face2](https://user-images.githubusercontent.com/42666898/62028940-f14ff400-b1fa-11e9-8b66-59bc92fcccc4.jpeg)
